lib/run.js

```diff
@@ -241,6 +241,7 @@ function _run (opts, cb, tracker) {
     function onError (error) {
       for (let i = 0; i < opts.pipelining; i++) tracker.emit('reqError', error)
       errors++
+      if (opts.debug) console.error(error)
```

Comment:
I would love to have a test that requires this and mocks `console.error` to verify things gets printed out.